### DISCUSSION TITLE
[Esame 15 gennaio 2024] Esercizio 3

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#452] [Esami M-Z]
![15 gen es.3](https://github.com/user-attachments/assets/0213c2be-0e69-4f5a-831d-eba565b4458a)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/daeddf0e-241c-4912-92f1-81251c7eaf24" alt="soluzione"  />
  <img src="https://github.com/user-attachments/assets/07c43a01-a45e-44a6-a039-b5afc3b670be" alt="soluzione 2"  />
  
</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/33014d63-331e-446a-846a-b04d283fe11d)
![immagine](https://github.com/user-attachments/assets/0eaad077-46d1-4397-95dc-2263ef43ac9a)

**Comment by niccolo-fato** [comment]
<img width="826" height="1280" alt="image" src="https://github.com/user-attachments/assets/871c30f1-bc44-410c-b3ce-7c7720d2efee" />
